modules/customizing-templates/con-about-software-templates.adoc

```diff
@@ -0,0 +1,9 @@
+[id='con-about-software-templates']
+= About Software Templates
```

Comment:
The content of this concept module is functionally replacing the assembly-configuring-templates introduction. Consider either removing the current assembly introduction, or replacing and expanding the assembly introduction with the content of this concept module.

Consider using "Software Templates" consistently in the entire section (assembly + included modules). For example the assembly title should be "Configuring Sotfware Templates" rather than "Configuring templates".